### ISSUE TITLE
How to pad Mobilenet v2 to run on KPU?

### ISSUE THREAD

**Issue by shanejohnpaul** [kendryte/nncase#166]
According to [this](https://github.com/kendryte/nncase/tree/f92c085ec6355ae04258df5b76ec7570f784129d) (at the end of the readme),

> * When using TensorFlow Conv2d/DepthwiseConv2d kernel=3x3 stride=2 padding=same, you must first use tf.pad([[0,0],[1,1],[1,1],[0,0]]) to pad the input and then use Conv2d/DepthwiseConv2d with valid padding.

But the current [FAQ](https://github.com/kendryte/nncase/blob/master/docs/FAQ_EN.md) under constraints says, 

> Same symmetric paddings (TensorFlow use asymmetric paddings when stride=2 and size is even).

And what does 'size' mean? I suppose it refers to the feature map. If so, only before those convolutions should we add padding or regardless of size should we pad the feature maps?

I found [this](https://github.com/sipeed/Maix-Keras-workspace/blob/master/mbnet/mobilenet.py) code for Mobilenet v1 which uses ZeroPadding2D to pad and it works. Is there a modified version of Mobilenet v2 from Keras available which already meets all the constraints?

**Comment by valentin7121** [comment]
The size means feature map height and width. If height or width is even and stride=2, we should add padding manualy and change padding type of Conv layer to 'valid'. For example:

```
x_pad = tf.pad(inputs, [[0,0],[1,1],[1,1],[0,0]])
x = layers.Conv2D((3, 3), padding='valid', strides=(2,2))(x_pad)
```
